packages/generate-persisted-query-manifest/cli.js

```diff
@@ -2,7 +2,11 @@
 
 const { Command } = require("commander");
 const { cosmiconfig } = require("cosmiconfig");
-const { generatePersistedQueryManifest, defaults } = require("./dist/index.js");
+const {
+  generatePersistedQueryManifest,
+  getFilepaths,
+  defaults,
+} = require("./dist/index.js");
```

Comment:
I'm [blaming you](https://github.com/apollographql/apollo-utils/pull/287/commits/f4a710dbe22bf1b579299e1438ac6cb45ec912ab#diff-2e820a95431b3426c91e69100ea3eaac46e88669f0a0fa3aa53d8e0c78fbd229R3) for that 😜. Though I probably should have improved this when I took over haha.

But ya, we could probably do something here to make this a tad better. Using `dist` here means that I have to rebuild the package anytime I make changes (including for tests), which is a real pain. I'll see if I can address this in separate PR.

Reply:
I mean I think you can just delete it and this might have been some sort of VSCode-added mistake?

Reply:
Perhaps I'm misunderstanding your ask here. All of these functions are used in this file, so I need to import them from somewhere (here is [`getFilepaths`](https://github.com/apollographql/apollo-utils/pull/344/files#diff-2e820a95431b3426c91e69100ea3eaac46e88669f0a0fa3aa53d8e0c78fbd229R36) in `cli.js` for example). If I update this to be `require('./index.js')`, the CLI ceases to function with the error:

```
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module './index.js'
Require stack:
- /Users/jerelmiller/code/apollo-utils/packages/generate-persisted-query-manifest/cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:119:18)
    at Object.<anonymous> (/Users/jerelmiller/code/apollo-utils/packages/generate-persisted-query-manifest/cli.js:9:5)
    at Module._compile (node:internal/modules/cjs/loader:1233:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/jerelmiller/code/apollo-utils/packages/generate-persisted-query-manifest/cli.js'
  ]
}

Node.js v20.4.0
```

Am I missing something obvious that you're asking here?

Reply:
OH. I missed that this is in fact a very special file — it's a JS file run directly, not a TS file.  I can't recall why I decided to do that (as you point out, it is my fault) though I suppose the file was somewhat smaller back then :)

It would be reasonable to see if we could switch away from having any JS files in the repo, or getting it back down to being basically one line long — but certainly not in this PR.

Reply:
Most definitely! I figured you probably knew something I didn't when you started the CLI, so I've left it as a `.js` file. I'll explore migrating this to a ts file :)